mathlibtools/leanproject.py

```diff
@@ -3,7 +3,6 @@
 from datetime import datetime
 from typing import Tuple, Optional
 from getpass import getpass
-
 from git.exc import GitCommandError # type: ignore
```

Comment:
```suggestion

from git.exc import GitCommandError # type: ignore
```

Reply:
At the bottom of this file there is an `if __name__ == "__main__"`. Can you add a comment there to indicate that this is also needed for PyInstaller?